package.json

```diff
@@ -53,10 +66,11 @@
     "eslint-plugin-jest": "^28.3.0",
     "eslint-plugin-jest-formatting": "^3.1.0",
     "eslint-plugin-prettier": "^5.1.3",
-    "jest": "^29.2.2",
+    "fetch-mock": "^10.1.1",
+    "jest": "^29.7.0",
     "prettier": "^3.2.5",
     "semantic-release": "^24.0.0",
-    "ts-jest": "^29.0.3",
+    "ts-jest": "^29.1.2",
     "typescript": "^5.0.4",
     "undici": "^6.7.1"
```

Comment:
This one should no longer be needed per mock removals.

Reply:
patched